borrowed/guile-json/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 
-GNC_ADD_SCHEME_TARGETS(guile-json
+SET(guile-json_DATA
   json.scm
   json/builder.scm
   json/parser.scm
   json/syntax.scm
 )
 
-# SET_DIST_LIST(guile-json_DIST CMakeLists.txt ${guile-json_DATA})
+INSTALL(FILES ${guile-json_DATA} DESTINATION ${CMAKE_INSTALL_DATADIR}/gnucash/guile-json)
```

Comment:
I'm having real problems trying to add guile-json into the appropriate folder where it can be accessed via `(use-modules (json))` - I think it's because I'm not setting the appropriate GUILE_DEPENDS in standard-reports. Can anyone help me set the right folders?